config-connector/solutions/projects/kpt/shared-vpc/README.md

```diff
@@ -0,0 +1,49 @@
+Shared VPC Network
+==================================================
+
+# NAME
+
+  shared-vpc
+
+# SYNOPSIS
+
+  Config Connector YAML files to create a VPC network inside a host project,
+  which can be consumed from within a service project.
+
+## Consumption
+
+  Using [kpt](https://googlecontainertools.github.io/kpt/):
+  ```
+  kpt pkg get https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/config-connector/solutions/projects/kpt/shared-vpc .
+  ```
+
+## Requirements
+
+  A working cluster with Config Connector installed.
+
+  The cnrm-system service account, which must have
+`roles/resourcemanager.projectCreator` in the target organization if service and
+host projects do not yet exist, or the `owner` role in the projects if they already exist.
+
```

Comment:
Should mention need `compute.organizations.enableXpnHost` permission or Compute Shared VPC Admin role